tests/integration/other/test_comment_commands.py

```diff
@@ -355,7 +328,8 @@ class CommentCommands(CommandTestCase):
         self.assertIs(comment_list['page'], 1)
         self.assertIs(comment_list['total_items'], 28)
```

Comment:
if you want to check that comment_list has a set of keys, use a set instead of a loop:
```
self.assertEqual(
    {'items', 'page', 'page_size', 'has_hidden_comments', 'total_items', 'total_pages'}, set(claim_list)
)
```

also, `self.assertIs()` is for checking instance equality, here you want to use regular equality check with `assertEqual` instead

Reply:
That's super useful, I had no idea that set(dict) returned the set of its keys.